src/tests.ts

```diff
@@ -99,6 +99,24 @@ export async function testDebugHandler(
     return;
   }
 
+  let debugType = null;
+
+  const cppTools = vscode.extensions.getExtension("ms-vscode.cpptools");
+  if (cppTools && cppTools.isActive) {
+    debugType = "cppdbg";
+  } else {
+    const codelldb = vscode.extensions.getExtension("vadimcn.vscode-lldb");
+    if (codelldb && codelldb.isActive) {
+      debugType = "lldb";
+    }
+  }
+
+  if (!debugType) {
+    vscode.window.showErrorMessage("No debugger extension found. Please install one and try again");
```

Comment:
Can you add a period at the end of this message?

Reply:
There is no period at the end in other error messages in this file. In other places in the code, there's no consistency in how this is handled. I can do this, but fixing it seems to be outside the scope of this PR.

https://github.com/mesonbuild/vscode-meson/blob/2d91eebf3beebfd9429e098d8f67cac1ab88ea7d/src/tests.ts#L54
https://github.com/mesonbuild/vscode-meson/blob/2d91eebf3beebfd9429e098d8f67cac1ab88ea7d/src/tests.ts#L97

Reply:
Yeah keep it as is then. Thanks for checking.